src/imp/openssl.rs

```diff
@@ -258,7 +270,10 @@ impl TlsConnector {
         if let Some(ref identity) = builder.identity {
             connector.set_certificate(&identity.0.cert)?;
             connector.set_private_key(&identity.0.pkey)?;
-            for cert in identity.0.chain.iter().rev() {
+            for cert in identity.0.chain.iter() {
```

Comment:
Why is this order being reversed?

Reply:
I don't know the details. Previous discussion: https://github.com/sfackler/rust-native-tls/pull/147#discussion_r367203922

Reply:
In https://github.com/sfackler/rust-native-tls/commit/05fb5e583be589ab63d9f83d986d095639f8ec44 you wrote:

> Reverse chain loading for openssl
> The stack is the reverse of what you might expect due to the way
PKCS12_parse is implemented, so we need to load it backwards.
> Closes #110

If `PKCS12_parse` is the cause, maybe it's better to reverse the chain in `from_pkcs12`?

https://github.com/sfackler/rust-native-tls/blob/82ae9a35f92b3e86f44ee5c7245aa991eb8e63d2/src/imp/openssl.rs#L161

Reply:
Added [`7b7afc2` (#209)](https://github.com/sfackler/rust-native-tls/pull/209/commits/7b7afc2b7b290f0b075a066cd61ab1a4815379dd)